includes/abstracts/abstract-wc-payment-token.php

```diff
@@ -16,7 +16,7 @@
  * @category	Abstract Class
  * @author		WooThemes
  */
- abstract class WC_Payment_Token implements WC_Data {
+ abstract class WC_Payment_Token {
```

Comment:
It should but will require a bit more changes then just extending the class to do it "properly" since the token stuff was written before the CRUD work started (for example, it has its own meta code). I'd like to do that in a separate PR, though.

This branch would also need to be merged in for 2.6 to get used by the token class in 2.6. Are we good with that?